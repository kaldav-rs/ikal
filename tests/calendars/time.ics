BEGIN:VCALENDAR
VERSION:2.0
PRODID:collective/icalendar
X-SOMETIME;VALUE=TIME:172010
END:VCALENDAR
