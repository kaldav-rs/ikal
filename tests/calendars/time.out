Ok(
    VCalendar {
        prodid: Text {
            params: {},
            text: "collective/icalendar",
        },
        version: Text {
            params: {},
            text: "2.0",
        },
        calscale: None,
        method: None,
        alarms: [],
        events: [],
        freebusy: [],
        journals: [],
        todo: [],
        timezones: [],
        x_prop: {
            "X-SOMETIME": ContentLine {
                params: {
                    "VALUE": "TIME",
                },
                value: "172010",
            },
        },
        iana_prop: {},
    },
)
